homeassistant/components/comelit/__init__.py

```diff
@@ -31,10 +55,34 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
 
 async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
```

Comment:
I think these functions could be one. If you make that superclass thing work, maybe you can just add a method to close the coordinator

Reply:
Done